src/components/containers/CreateToken.container.tsx

```diff
@@ -71,7 +71,7 @@ export const getTextButtonByNetwork = (network: Network) => `Create API Token fo
 export const getTextButton = () => 'Create API Token'
 const getTextCreateTokenButton = (network: Network) =>
   ifElse(isActiveToggleNetwork, getTextButtonByNetwork, getTextButton)(network)
-const getNetworkByFT = (network: Network) => ifElse(isActiveToggleNetwork, identity, () => undefined)(network)
```

Comment:
Doesn't make much of a difference but maybe using [always](https://ramdajs.com/docs/#always) instead of `(): any => undefined`...?